### ISSUE TITLE
Crash during destroy

### ISSUE THREAD

**Issue by rklyne** [hashicorp/terraform#5743]
Crash log attached.
I was on a train on 3G at the time so a flaky network is probably part of the cause.

[crash.zip](https://github.com/hashicorp/terraform/files/181834/crash.zip)

**Comment by jen20** [comment]
Relevant part of crash log:

```
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: panic: runtime error: invalid memory address or nil pointer dereference
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: [signal 0xb code=0x1 addr=0x0 pc=0x61063]
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: 
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: goroutine 79 [running]:
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: panic(0x666080, 0x820bcc110)
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: 	/opt/go/src/runtime/panic.go:464 +0x3e6
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: github.com/hashicorp/terraform/builtin/providers/digitalocean.resourceDigitalOceanDropletRead(0x820e47da0, 0x69e740, 0x820e74ea0, 0x0, 0x0)
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: 	/opt/gopath/src/github.com/hashicorp/terraform/builtin/providers/digitalocean/resource_digitalocean_droplet.go:192 +0x263
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: github.com/hashicorp/terraform/helper/schema.(*Resource).Refresh(0x820eca8c0, 0x820ecd4a0, 0x69e740, 0x820e74ea0, 0x820ed5a90, 0x0, 0x0)
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: 	/opt/gopath/src/github.com/hashicorp/terraform/helper/schema/resource.go:209 +0x43a
2016/03/21 08:34:17 [DEBUG] terraform-provider-digitalocean: github.com/hashicorp/terraform/helper/schema.(*Provider).Refresh(0x820ecc630, 0x820ecb580, 0x820ecd4a0, 0x17b301, 0x0, 0x0)
```

**Comment by jen20** [comment]
Hi @rklyne - from the log it looks like you are using Terraform 0.6.12 - this should have been fixed in Terraform 0.6.13 as part of #5588. Thanks for reporting - if you experience this again with the latest version of Terraform please feel free to re-open!
